### ISSUE TITLE
 DataLoader worker (pid(s) 18056, 20540, 4512) exited unexpectedly

### ISSUE THREAD

**Issue by Aidenfaustine** [pytorch/pytorch#59438]
## ❓  DataLoader worker (pid(s) 18056, 20540, 4512) exited unexpectedly

I'm new to pytorch. when i run commands in Jupyternotebook  %run main.py --task wilder --emo_dim arousal --feature_set egemaps --normalize --norm_opts y --d_rnn 128 --rnn_n_layers 4 --epochs 100 --batch_size 1024 --n_seeds 10 --win_len 200 --hop_len 100 --use_gpu --cache --save --save_path preds

it reported error as follows:
DataLoader worker (pid(s) 18056, 20540, 4512) exited unexpectedly

Thanks
best wishes.
jiachen

**Comment by ejguan** [comment]
Can you please share a minimum script and your den env?
In lots of cases, interactive python interpreter, which notebook is using, doesn't work correctly with multiprocessing when you set num_workers > 0. Can you try to run your code our of notebook?
